app/javascript/react/components/ReviewTile.js

```diff
@@ -1,22 +1,21 @@
-import React, { Fragment } from 'react'
-import { useLocation } from 'react-router'
+import React, { Fragment } from "react";
+import { useLocation } from "react-router";
 
-const ReviewTile = props => {
+const ReviewTile = (props) => {
   let coffeeRating = [];
-  for(let i=0; i < props.rating; i++){
-    coffeeRating.push(<i className="fa fa-coffee" aria-hidden="true"></i>)
+  for (let i = 0; i < props.rating; i++) {
+    coffeeRating.push(<i className="fa fa-coffee" aria-hidden="true"></i>);
   }
-  
+
   return (
-      <div>
-        <li>
-            <span>Rating: </span>
-            {coffeeRating}
-            <p>{props.comment}</p>
-            <p className="user-review-post">Posted By: GuestUser123 </p>
-        </li>
-      </div>
-  )
-}
+    <div>
```

Comment:
I wouldn't add this `<div>` here, it's ok to just have `<li>`. The reason being is now inside of your `<ul>` you are also going to have `<div>` and `<li>`
like so:

```
<ul>
  <div>
      <li> stuff</li>
   </div>   
   
     <div>
      <li> stuff</li>
    </div>   
   
     <div>
      <li> stuff</li>
    </div>   
</ul>
```

That could cause some issues when you add more styling.